### PR TITLE
[SMALLFIX] Make more top-level git-ignored dirs specific

### PR DESCRIPTION
**Pull request by apc999**
and reorder the git-ignored items alphabetically

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,25 +11,24 @@
...
+/data/
+/docs/_site/
+/docs/api/
+/docs/serve/
 /journal/
-conf/tachyon-env.sh
-conf/tachyon-env.sh.bootstrap.bk
-data/
 dependency-reduced-pom.xml
+deploy/vagrant/.request_ids
```
Comment (uronce-cc) [inline .gitignore]: `/deploy/....`

File: .gitignore [hunk 2]
```diff
@@ -10,25 +11,24 @@
...
 deploy/vagrant/spot/roles/tag_ec2/vars/
-deploy/vagrant/bin/*.pyc
-deploy/vagrant/.request_ids
-docs/_site
-docs/api
-docs/serve/
+deploy/vagrant/vbox/.vagrant/
+deploy/vagrant/vbox/tachyon-dev.box
 logs/
 script/
```
Comment (uronce-cc) [inline .gitignore]: Seems like no use.

File: .gitignore [hunk 3]
```diff
@@ -1,8 +1,9 @@
 # TODO(gene): Use leading '/' to specifically ignore top level directories.
```
Comment (uronce-cc) [inline .gitignore]: remove?

**Comment by apc999** [comment]
Jenkins, test this please.

File: .gitignore (hunk 3 above)
Comment (apc999) [inline .gitignore]: Done

File: .gitignore (hunk 1 above)
Comment (apc999) [inline .gitignore]: done

File: .gitignore (hunk 2 above)
Comment (apc999) [inline .gitignore]: Removed

**Comment by apc999** [comment]
@uronce-cc  PTAL. Thanks.

**Comment by yupeng9** [comment]
LGTM

**Comment by yupeng9** [comment]
Thanks for the cleanup.

**Comment by uronce-cc** [comment]
LGTM

**Pull request merged**